runtime/doc/lsp.txt

```diff
@@ -65,26 +65,75 @@ Follow these steps to get LSP features:
 5. (Optional) Configure keymaps and autocommands to use LSP features.
    |lsp-attach|
 
-                                                        *lsp-config*
+==============================================================================
+DEFAULTS                                                *lsp-defaults*
```

Comment:
- lsp-defaults now has a section, and immediately follows lsp-quickstart.
- lsp-config has a section after that, introducing "static config" (vim.lsp.config) and "dynamic config" (LspAttach/on_attach).